src/components/ui/Markdown.tsx

```diff
@@ -1,22 +1,38 @@
-import clsx from "clsx";
 import { type ComponentProps } from "react";
 import ReactMarkdown from "react-markdown";
 import { Link } from "./Link";
+import { tv } from "tailwind-variants";
+
+const markdownVariants = tv({
```

Comment:
Let's keep the Markdown and Card components as is and style from the Metrics components instead.

The components in the UI folder should be generic and not reference to any app-specific things.

Reply:
noted